src/ChebyshevApproximation.cc

```diff
@@ -128,18 +119,17 @@ ChebyshevApproximation::computeChebyshevApproximation()
 {
     compute_tm_.start();
 
-    assert(nodesTk_ != 0);
-    assert((int)coeffs_.size() == order_);
-    //    assert((int)nodesTk_->size() == order_);
+    assert(nodesTk_ != nullptr);
```

Comment:
`nodesTK_` is not a pointer any more so I guess you want to check that the size is non-zero

Reply:
Good catch! Thanks. I also see that some CI tests failed, which is weird. I'll look into that and push an update.

Reply:
That's the failure from the CI.

Reply:
Ah, I see that now. Thanks. I pushed the update.